src/gluonts/ev/ev_tutorial.md

```diff
@@ -0,0 +1,162 @@
+# New evaluation approach
+
+This notebook explains how to use the `ev` module which is a new alternative to the `evaluation` module in GluonTS.
+
+## [Usage](usage_example.py)
+
+Before diving into how things work, let's look at how to use the new evaluation approach.
+
+### Overview
+1. Decide on the `test_data` (of type `TestData`) and `predictor` to use.
+2. Gather the `metrics` to be evaluated and decide over what `axis` to aggregate (use `None` to aggregate to a single value).
+3. Call `predictor.backtest(metrics, test_data, axis)` to get (metric name, metric result) pairs.
+
+### Example
```

Comment:
This example is way too complicated and does too many things. It suffers from the same problem as our other tutorials, it tries to do too much. Let's focus on one aspect of the evaluation and not things like splitting into pairs.

Reply:
Good idea. 👍 

Using dummy data, I was able to avoid using `TestData` in this [commit](https://github.com/awslabs/gluonts/pull/2267/commits/bc583c386e0a97c869eb5264a32df282b0c48609).

I think, this makes the tutorial more focused on the `ev` module and is easier to understand without having to know how `TestData` works.